### PR TITLE
release-controller: test new periodics functionality

### PR DESCRIPTION
**Pull request by AlexNPavel**
This PR tests the new release-controller periodic functionality by
enabling it for 2 periodic release informers. If everything behaves as
expected, a future PR will migrate all self-annotating release informers
to release-controller periodics.

/cc @stevekuznetsov

### CODE REVIEW

**Comment by AlexNPavel** [comment]
/hold

Requires https://github.com/openshift/release-controller/pull/217 before merging

**Review by stevekuznetsov — APPROVED** [review]

**Pull request merged**
